.azure-pipelines/official-build/docker-sonic-mgmt.yml

```diff
@@ -35,7 +35,7 @@ stages:
     pool: sonictest
     timeoutInMinutes: 360
     steps:
-    - template: cleanup.yml
+    - template: ../cleanup.yml
```

Comment:
This change is nothing to do the cache build, and it will break the azure pipeline for docker-sonic-mgmt, better to add in another PR.